graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/fetchers/deployed/QueryFetcher.java

```diff
@@ -40,7 +40,7 @@ public class QueryFetcher extends DeployedFetcher<Object> {
 
   @SuppressWarnings("unchecked")
   private static final Coercing<ByteBuffer, String> BLOB_COERCING =
-      CqlScalar.BLOB.getGraphqlType().getCoercing();
+      (Coercing<ByteBuffer, String>) CqlScalar.BLOB.getGraphqlType().getCoercing();
```

Comment:
This was necessary because of backwards-incompatible change where `getGraphqlType()` now returns "raw" type -- `Coercing<?,?>` -- instead actual types. I don't know why, and did not see mention in release notes (but there were many things so could have missed).
So I only changed it because this (or something else) is necessary due to API change. :-(

Maybe @olim7t has more context.